packages/path_provider/path_provider_linux/lib/path_provider_linux.dart

```diff
@@ -3,11 +3,21 @@
 // found in the LICENSE file.
 import 'dart:io';
 import 'dart:async';
+import 'dart:ffi';
```

Comment:
As I recently learned the hard way, if you add FFI to a plugin that's manually registered (as all desktop Dart plugins currently have to be) you need to do [this](https://github.com/flutter/plugins/pull/3056) to avoid breaking web builds. This won't show up in CI currently, unfortunately.